Makefile

```diff
@@ -14,8 +14,6 @@
 # to check the official mod dlls for StyleCop violations, run:
 #   make check
 #
-# to generate documentation aimed at modders, run:
-#   make docs
 #
```

Comment:
Sorry, this is really nit-picky, but can you remove one of these two empty lines as well, please?